### PR TITLE
Papers on various execution approaches

### PR DESCRIPTION
**Pull request by smarr**
This is rapidly getting less and less structured, I fear.

### CODE REVIEW

File: graph.dot [hunk 1]
```diff
@@ -71,4 +132,32 @@ digraph G {
...
+  exe -> maps [label = "method JIT"]
+  exe -> regionBased [label = "region JIT"]
+  exe -> traceBased [label = "trace JIT"]
+  exe -> basicBlockVersioning [label = "basic-block JIT"]
+  basicBlockVersioning -> shapesAndBBV [label = "type propagation"]
+  traceBased -> traceMonkey
+  maps -> adaptiveJit [label = "+ adaptive compilation"]
+
+  traceBased -> metaTracing [label = "+ reusable JOT"]
+  traceMonkey -> metaTracing [label = "+ reusable JOT"]
```
Comment (eregon) [inline graph.dot]: JOT -> JIT?

**Comment by eregon** [comment]
> This is rapidly getting less and less structured, I fear.

Do you mean about the rendering/layout, or something else?

File: graph.dot (hunk 1 above)
Comment (smarr) [reply graph.dot]: Fixed

**Comment by smarr** [comment]
Well, layout but also the question of what's displayed.

There are multiple interesting dimensions, I think.
There are the papers themselves, and how they influenced each other.
And there are the specific concepts. The SELF paper for instance, has multiple important ideas.
So, I was a bit torn on what the nodes should be, concepts or papers.
I may have changed your original thinking, and turned it into a paper-based view, instead of a concept based view.

**Comment by eregon** [comment]
> and turned it into a paper-based view, instead of a concept based view.

That would be fine, I used the concepts/ideas (oval nodes) only when I didn't know a good paper to put at the root of that subtree.

**Comment by eregon** [comment]
Looks OK to me :)
https://raw.githubusercontent.com/eregon/virtual-machines-research-graph/master/graph.svg?sanitize=true

**Pull request merged**